lib/beaker/host.rb

```diff
@@ -324,7 +324,8 @@ def exec command, options={}
           # exit codes at the host level and then raising...
           # is it necessary to break execution??
           if options[:accept_all_exit_codes] && options[:acceptable_exit_codes]
-            @logger.warn ":accept_all_exit_codes & :acceptable_exit_codes set. :accept_all_exit_codes overrides, but they shouldn't both be set at once"
+            @logger.warn ":accept_all_exit_codes & :acceptable_exit_codes set. :acceptable_exit_codes overrides, but they shouldn't both be set at once"
```

Comment:
oh.  sorry, i guess i missed it.  it could probably use a re-word, but i wouldn't hold the PR up for it